api/Makefile

```diff
@@ -48,13 +48,13 @@ test:
 	# Run tests
 	# useful params for pytest:
 	# -s show stdout
-	# -n numprocesses - doesn't really seem to run faster with more proccesses at the moment.
-	PYTHONPYCACHEPREFIX=python_cache $(POETRY_RUN) pytest app;
+	# -n numprocesses
+	PYTHONPYCACHEPREFIX=python_cache $(POETRY_RUN) pytest -n 3 app;
 	# ImportMismatchError? run: make clean
 
 test-verbose:
 	# Run tests with verbose output
-	$(POETRY_RUN) pytest app -vv;
+	$(POETRY_RUN) pytest app -n 2 -vv;
```

Comment:
Coverage doesn't actually have -n ; coverage gets confused when you're running in parallel.

I'll change it to -n 3 ; 3 is just what I've found runs fastest for me. 1 (or not specifying parallel) has now just gotten too slow because of my c-haines test.